xclim/indicators/atmos/_temperature.py

```diff
@@ -834,11 +839,18 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
 
 warm_spell_duration_index = Temp(
     identifier="warm_spell_duration_index",
-    description="{freq} total number of days within spells of at least {window} days"
-    " with tmax above the 90th daily percentile.",
+    description="{freq} number of days with at least {window} consecutive days "
+    "where the daily maximum temperature is above the {per_base_thresh}th "
+    "percentile(s). A {per_window} day(s) window, centred on each calendar day in the "
+    "{per_period} period, is used to compute the {per_base_thresh}th percentile(s).",
```

Comment:
I thought about it and I agree, but without a proper i18n module, including for English, it will be very artisanal to do so and might not work well in French (and upcoming translations).
Still I can go the artisanal way.